cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -45,12 +83,99 @@ class CoursewareSearchIndexer(object):
     Class to perform indexing for courseware search from different modulestores
     """
 
+    # Flags for where to index the information
```

Comment:
Maybe just make an additional note here about the use of bitwise programming for future collaborators